### PR TITLE
Authentication type none added & Copy to Clipboard on double cli…

### PR DESCRIPTION
**Pull request by SouraDutta**
1. Authentication type None has been introduced.
    - This will help to bypass double layer of Passwords required for SSH.
    - This allows to SSH without any authentication (Weakest type).

2. This is to mimic the behavior of select to copy on double click feature of Unix terminals.

### CODE REVIEW

File: src/client/index.ts [hunk 1]
```diff
@@ -17,12 +17,36 @@ const socket = io(window.location.origin, {
 const FILE_BEGIN = '\u001b[5i';
 const FILE_END = '\u001b[4i';
 
+//NOTE text selection on double click or select
```
Comment (butlerx) [inline src/client/index.ts]: Can you put this function in a separate file and import it

File: src/server/command.ts [hunk 2]
```diff
@@ -68,13 +68,18 @@ function sshOptions(
     port,
     '-o',
     `PreferredAuthentications=${auth}`,
-  ];
+    ];
+  console.log(`Authentication Type: ${auth}`)
```
Comment (butlerx) [inline src/server/command.ts]: please dont use `console.log` import the logger and use info logging

**Review by butlerx — CHANGES_REQUESTED** [review]
Could you please increment the the package version and run `yarn lint --fix` and fix any errors

File: src/client/index.ts (hunk 1 above)
Comment (mikr13) [reply src/client/index.ts]: Thanks for this great suggestion but then we are looking at passing **term** (terminal) object to that file, will that be okay?
Accessed here:
```
   if (term.hasSelection()) 
      copyToClipboard(term.getSelection())
```

**Comment by mikr13** [comment]
> Could you please increment the the package version and run `yarn lint --fix` and fix any errors

I guess it's already being done by husky before commit, is it required to be done manually also?

File: src/server/command.ts (hunk 2 above)
Comment (mikr13) [reply src/server/command.ts]: Fixed in latest commit

File: src/client/index.ts (hunk 1 above)
Comment (butlerx) [reply src/client/index.ts]: you can leave everything in the `socker.on("connect")` as is but move `copyToClipboard` to an external file and export it. and then import it at the top of this file as `import copyToClipboard from "./clipboard";`

**Comment by butlerx** [comment]
hmm I wonder why the spacing changed. if husky ran youre good

File: src/client/index.ts (hunk 1 above)
Comment (mikr13) [reply src/client/index.ts]: Okay, so you are saying only the function should be in different file? Yeah that can be done, we'll do that. Thanks.

**Comment by mikr13** [comment]
> hmm I wonder why the spacing changed. if husky ran youre good

Yeah it ran and there were no errors but I'll like to tell you that the plugin we are using is deprecated and there are newer alternatives for it.

File: src/client/index.ts (hunk 1 above)
Comment (mikr13) [reply src/client/index.ts]: In the latest update

- Copy To clipboard function separated

- Yarn lint is done

- Version number bumped from 1.1.8 to 1.1.9

File: .eslintrc.js [hunk 3]
```diff
@@ -20,6 +20,7 @@ module.exports = {
     'func-style': ['error', 'declaration', { allowArrowFunctions: true }],
     'no-use-before-define': ['error', { functions: false }],
     '@typescript-eslint/no-use-before-define': ['error', { functions: false }],
+    'consistent-return': 1
```
Comment (butlerx) [inline .eslintrc.js]: id rather now disable this.

File: src/client/copyToClipboard.ts [hunk 4]
```diff
@@ -0,0 +1,22 @@
+// NOTE text selection on double click or select
+const copyToClipboard = (text: string) : any => {
```
Comment (butlerx) [inline src/client/copyToClipboard.ts]: can we make this a consistent type

**Review by butlerx — CHANGES_REQUESTED** [review]

File: src/client/copyToClipboard.ts (hunk 4 above)
Comment (mikr13) [reply src/client/copyToClipboard.ts]: Yup, done in latest commit.

**Review by butlerx — APPROVED** [review]

**Comment by mikr13** [comment]
@butlerx Actually myself @MiKr13 & @SouraDutta both Worked on these features but only one of us are showing as contributors, can you explain this behaviour as we don't have idea about this.

**Comment by butlerx** [comment]
im unsure why you are not marked as a contributor. If youd like to, you can add yourself to the contributor list in package.json

**Comment by mikr13** [comment]
@butlerx sure, we'll do that.

**Pull request merged**